pkg/resource/deploy/step.go

```diff
@@ -353,7 +353,7 @@ type UpdateStep struct {
 	reg           RegisterResourceEvent          // the registration intent to convey a URN back to.
 	old           *resource.State                // the state of the existing resource.
 	new           *resource.State                // the newly computed state of the resource after updating.
-	stables       []resource.PropertyKey         // an optional list of properties that won't change during this update.
+	stables       []string                       // an optional list of properties that won't change during this update.
```

Comment:
why did it switch to string?

Reply:
They're no longer simple keys; they're now paths.

Reply:
looks like it's a path now?

1. consider stating that in the comment.
2. should we consider a typedef for something like `PropertyPath` to make it clearer what you're operating on?